src/python/grapl_analyzerlib/test2.py

```diff
@@ -1,13 +1,9 @@
 from typing import *
```

Comment:
@colin-grapl do we even care about this poorly named test class? wtf is aiud

Reply:
We do not. auid refers to `audit user id` though, something that does not exist today.